src/main/java/org/opentripplanner/routing/api/request/request/VehicleParkingRequest.java

```diff
@@ -1,45 +1,68 @@
 package org.opentripplanner.routing.api.request.request;
 
 import java.io.Serializable;
-import java.util.HashSet;
-import java.util.Set;
+import org.opentripplanner.routing.api.request.request.filter.VehicleParkingFilterRequest;
 
-// TODO VIA: Javadoc
+/**
+ * Class that stores information about what kind of parking lots should be used for Park & Ride
+ * and Bike & Ride searches.
+ */
 public class VehicleParkingRequest implements Cloneable, Serializable {
```

Comment:
We should make the request class immutable similar to how the preferences are.

Reply:
I would like to do that in a separate PR.

Should this be created through a builder?